stdlib/public/core/BidirectionalCollection.swift

```diff
@@ -239,8 +241,8 @@ extension BidirectionalCollection where SubSequence == Self {
   ///   elements in the collection.
   ///
   /// - Complexity: O(1) if the collection conforms to
-  ///   `RandomAccessCollection`; otherwise, O(*n*), where *n* is the length
-  ///   of the collection.
+  ///   `RandomAccessCollection`; otherwise, O(*n*), where *n* is the number of
+  ///   elements to remove.
```

Comment:
Shouldn't the *n* be *k* under the rule you stated?

Reply:
Thanks for the review, @brentdax! Fixed these.